src/templates/_macros/feed_item.html

```diff
@@ -145,20 +106,22 @@ <h1 class="name">{{ feed_app.app.name|translate(feed_app) }}</h1>
         {% endfor %}
       </ul>
     {% endif %}
-    <div class="curve" style="background-color: {{ inline_color|hex2rgba('.55') }}"></div>
-    <div class="info" style="background-color: {{ inline_color|hex2rgba('.55') }}">
+    <div class="feed-curve curve" style="background-color: {{ coll.inline_color|hex2rgba('.55') }}"></div>
+    <div class="info" style="background-color: {{ coll.inline_color|hex2rgba('.55') }}">
       <h1 class="name">{{ coll.name|translate(coll)|safe }}</h1>
     </div>
-    <div class="alt-curve" style="background-color: {{ inline_color }}"></div>
+    <div class="alt-curve" style="background-color: {{ coll.inline_color }}"></div>
```

Comment:
You'll hit some collisions here with my commit that already landed. Should be easy though.